infra/aws/terraform/kops-infra-ci/eks.tf

```diff
@@ -132,21 +131,20 @@ module "eks" {
       }
 
       tags = merge(var.tags, {
-        "region" = "${data.aws_region.current.name}"
+        "region" = local.region
```

Comment:
I am curious to learn more about the use of a region tag - is this used by something?

Reply:
Not really. some of tests pick regions randomly during their run. we want to be able to track cost per region.

Reply:
costs are tracked via AWS CUR (Cost and Usage Reporting), correct? Region is a very common facet that is carried over in nearly everything, including CUR reports https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html#product-details-R